### PR TITLE
Update contributing companies with latest status

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Comment by mraible** [comment]
@jdubois @pascalgrimaud Do you know if the statement about Ippon is still accurate? If not, we should update it.

**Comment by pascalgrimaud** [comment]
@mraible : I removed Ippon here https://github.com/jhipster/jhipster.github.io/pull/1274 but I probably forgot this part

**Comment by mshima** [comment]
Should we add a past contributors?
No company maintaining any generator anymore.

File: docs/contributing/contributing-companies.mdx [hunk 1]
```diff
@@ -7,9 +7,9 @@ last_update:
 
 ## Companies contributing to JHipster
 
-* [Ippon Technologies](https://fr.ippon.tech/), which provides 1 day per month to contribute for everyone who is in the Ippon JHipster team.
+* [Ippon Technologies](https://fr.ippon.tech/), which provides one day per month to contribute for everyone who is in the Ippon JHipster team.
 * [Microsoft](https://www.microsoft.com/), which provides time and hardware to [@juliendubois](https://x.com/juliendubois) to improve the project.
```
Comment (pascalgrimaud) [inline docs/contributing/contributing-companies.mdx:11]:
```suggestion
* [Microsoft](https://www.microsoft.com/), which provides time and hardware to [@juliendubois](https://x.com/juliendubois) to improve the project.
```

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Comment by mraible** [comment]
@SvMak Why does the bottom of the page say it was last updated in 2017 if it was updated today?

image

**Comment by SvMak** [comment]
> @SvMak Why does the bottom of the page say it was last updated in 2017 if it was updated today?
> 
> image

@mraible Because:
Screenshot 2024-08-29 at 22 02 57

You can try to delete this and get the time during the build, but there may be troubles, also the time can be specified in the format YYYY-MM-DD, for example: 2017-11-29

**Comment by mraible** [comment]
That makes sense. Thanks. 🙏

**Pull request merged**